pkg/engine/validation.go

```diff
@@ -647,19 +648,33 @@ func (v *validator) validatePatterns(resource unstructured.Unstructured) *respon
 			}
 
 			if pe, ok := err.(*validate.PatternError); ok {
-				v.log.V(3).Info("validation rule failed", "anyPattern[%d]", idx, "path", pe.Path)
-				if pe.Path == "" {
-					patternErr := fmt.Errorf("rule %s[%d] failed: %s", v.rule.Name, idx, err.Error())
-					failedAnyPatternsErrors = append(failedAnyPatternsErrors, patternErr)
+				var patternErr error
+				v.log.V(3).Info("validation error", "anyPattern[%d]", idx, "path", pe.Path)
```

Comment:
why change the error message? also instead of `Info` it should be `Error` right?

Reply:
Reverted back to original message. And the log level is `Info` in this case as they followed the same pattern in line 611 same file.